content/zh-cn/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -707,12 +728,14 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `APIResponseCompression`: Compress the API responses for `LIST` or `GET` requests.
 - `APIServerIdentity`: Assign each API server an ID in a cluster.
 - `APIServerTracing`: Add support for distributed tracing in the API server.
+  See [Traces for Kubernetes System Components](/docs/concepts/cluster-administration/traces-for-kubernetes-system-components) for more details.
 -->
 - `APIListChunking`：启用 API 客户端以块的形式从 API 服务器检索（“LIST” 或 “GET”）资源。
-- `APIPriorityAndFairness`: 在每个服务器上启用优先级和公平性来管理请求并发。（由 `RequestManagement` 重命名而来）
+- `APIPriorityAndFairness`: 在每个服务器上启用优先级和公平性来管理请求并发（由 `RequestManagement` 重命名而来）。
 - `APIResponseCompression`：压缩 “LIST” 或 “GET” 请求的 API 响应。
 - `APIServerIdentity`：为集群中的每个 API 服务器赋予一个 ID。
 - `APIServerTracing`: 为集群中的每个 API 服务器添加对分布式跟踪的支持。
+  参阅[针对 Kubernetes 系统组件的追踪](/docs/concepts/cluster-administration/traces-for-kubernetes-system-components)获取更多详细信息。
```

Comment:
```suggestion
  参阅[针对 Kubernetes 系统组件的追踪](/zh-cn/docs/concepts/cluster-administration/traces-for-kubernetes-system-components)获取更多详细信息。
```

Reply:
This en link is broken with 404, reported at issue #36280.

Reply:
Should be `/zh-cn/docs/concepts/cluster-administration/system-traces/`